inst/scripts/generateHGNCHash.R

```diff
@@ -0,0 +1,59 @@
+# generateHGNCHash.R
+
+#' Generates a data frame containing all valid HGNC gene symbols, calls fmatch
+#' on the data frame so it will create a hash of the data frame to be used by
+#' checkGeneSymbols.R.
+#'
+#' @param fURL The URL to the file containing valid HGNC gene symbols.
+#'
+#' @seealso \code{\link{isGeneSymbol}} is used to check if a gene symbol is a
+#' valid HGNC gene symbol that is contained in inst/extdata/HGNCSymbols.rds.
+
+library(fastmatch)
+
+# Open the connection/file.
+## Change fURL to be the URL of the file containing HGNC gene symbols.
+## The file is assumed to contain the symbol, status and locus group of the gene.
+fURL <- "http://tinyurl.com/kk3r9ck"
```

Comment:
done, I think (If I understood what you mean by **actile** URL correctly).
Could you clarify what you mean by an **actile** URL please?